fastai2/medical/imaging.py

```diff
@@ -34,7 +41,10 @@ def pixels(self:DcmDataset):
 def scaled_px(self:DcmDataset):
     "`pixels` scaled by `RescaleSlope` and `RescaleIntercept`"
     img = self.pixels
-    return img*self.RescaleSlope + self.RescaleIntercept
+    if self.Modality == "CR":
```

Comment:
This test can fit in one line: 
`return img if bla else ...`
Would you mind adjusting your PR with this?